src/CSSBattle/95_Pokeball/ZhangZhikai.html

```diff
@@ -0,0 +1,74 @@
+
+<div class="rectbehind">
+</div>
+<div class="semi">  </div>
+<div class="topblock"></div>
+
+<div class="semi bot">  </div>
+<div class="botblock"></div>
+<div class="rectbehind">
+</div>
+ <div class="center"></div>
+
+
+
+
+
+
```

Comment:
Consider removing unnecessary line breaks and moving `<div></div>` onto the same line

Reply:
50ed92b92dcb6d7fac9a9b6965067d8e1abde559